packages/rum/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { Datacenter } from '@datadog/browser-core'
+export { Datacenter, ErrorSource } from '@datadog/browser-core'
```

Comment:
should not we also exposed the `ManuallyAddedErrorSource` type used by addError?
from a user perspective it is a mouthful, it could be nicer to either:
- rename it to ErrorSource and prefix the core one with internal
- rename it to RumErrorSource